tests/mockserver_tests/mock_server_test_base.py

```diff
@@ -61,6 +61,27 @@ def aborted_status() -> _Status:
 def unavailable_status() -> _Status:
     error = status_pb2.Status(
         code=code_pb2.UNAVAILABLE,
+        message="Received unexpected EOS on DATA frame from server",
+    )
+    retry_info = RetryInfo(retry_delay=Duration(seconds=0, nanos=1))
+    status = _Status(
+        code=code_to_grpc_status_code(error.code),
+        details=error.message,
+        trailing_metadata=(
+            ("grpc-status-details-bin", error.SerializeToString()),
+            (
+                "google.rpc.retryinfo-bin",
+                retry_info.SerializeToString(),
+            ),
+        ),
+    )
+    return status
+
+
+# Creates an INTERNAL status with the smallest possible retry delay.
+def internal_status() -> _Status:
```

Comment:
INTERNAL errors normally do not include any retry info.

Reply:
Roger that and thanks. Done!